packages/engine-server/src/markdown/utils.ts

```diff
@@ -631,6 +513,128 @@ export class MDUtilsV4 {
   }
 }
 
+/** Contains functions that help dealing with mdast trees. */
```

Comment:
Done!

Agreed, maybe after `'MDUtilsV4` is fully deprecated we can rename `MDUtilsV5` to something like `MDProcessor`, since it only houses the processor now.